lib/OpenQA/File.pm

```diff
@@ -79,8 +76,7 @@ sub get_piece {
     return $self->_chunk($index, $n_chunks, $chunk_size, $residual, $total_cksum);
 }
 
-sub split {
-    my ($self, $chunk_size) = @_;
+sub split ($self, $chunk_size = undef) {
```

Comment:
Shouldn't this one be `$chunk_size = 10000000`?

Reply:
Signatures have one problem that can be easily overlooked: the default value will only be assigned if there was no value passed, e.g. if the argument list was empty her e(except `$self`).
So if someone passes undef by passing a variable, e.g.
`$file->split($some_var_that_might_be_undef)`
then the default assignment won't happen.

So passing `undef` here is the safest way.

Reply:
True, then I could just delete the other line